wordle-web/src/components/AppBar.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-card class="mx-auto" max-width="448" zIndex="0">
+    <v-layout>
+      <v-app-bar density="compact">
+        <template v-slot:prepend>
+          <v-btn icon="mdi-menu" @click.stop="hamburgerDrawer = !hamburgerDrawer"> </v-btn>
+        </template>
+        <v-icon icon="mdi-head-cog" @click.stop="$router.push('/')" size="x-large"></v-icon>
+        <v-app-bar-title @click.stop="$router.push('/')">Wordle</v-app-bar-title>
+        <template v-slot:append>
+          <v-btn icon="mdi-cog" @click.stop="settingDrawer = !settingDrawer"> </v-btn>
+        </template>
+      </v-app-bar>
+
+      <v-navigation-drawer temporary v-model="hamburgerDrawer" location="left">
+      </v-navigation-drawer>
+
+      <v-navigation-drawer v-model="settingDrawer" location="right">
+        <v-list>
+          <v-btn @click="setInverseTheme"> Inverse Theme </v-btn>
+          <v-btn @click="setDarkTheme"> Dark Theme </v-btn>
+        </v-list>
+      </v-navigation-drawer>
```

Comment:
If you are having trouble with these drawers working I think the Vuetify menu component would also be perfectly acceptable.

Reply:
Finalized in commit [df707cb](https://github.com/IntelliTect-Samples/EWU-CSCD379-2023-Spring/pull/50/commits/df707cba648015ab0160d1ecd0fc9a349d22e8f2)